libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid-wrapper.component.ts

```diff
@@ -80,19 +85,25 @@ export class SkyAgGridWrapperComponent
   #currentTheme: SkyThemeSettings | undefined = undefined;
   #adapterService: SkyAgGridAdapterService;
   #changeDetector: ChangeDetectorRef;
+  #parentChangeDetector: ChangeDetectorRef | undefined;
   #elementRef: ElementRef;
   #document: Document;
 
   constructor(
     adapterService: SkyAgGridAdapterService,
     changeDetector: ChangeDetectorRef,
+    @Optional()
+    @SkipSelf()
+    @Inject(ChangeDetectorRef)
+    parentChangeDetector: ChangeDetectorRef | undefined,
```

Comment:
Could we place this new parameter at the end of the constructor and use `?` instead of `| undefined`?

Reply:
Updated.